### PR TITLE
Update New-CsOnlineSession.md

### PR DESCRIPTION
**Pull request by pupara**
Removed UserName. Made Credential as default.

### CODE REVIEW

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 1]
```diff
@@ -69,6 +64,8 @@ Establishes a Skype for Business Online Remote PowerShell session using multi-fa
 
 ### -UserName
 Specifies the Skype for Business Online administrator account name to be used when prompting for credentials.
+> [!WARNING]
+> UserName has been deprecated and is not avaiable in Teams Powershell module. Please use Credential parameter to provide username and password.
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsOnlineSession.md:68]:
```suggestion
Please Note: UserName has been deprecated and is not avaiable in Teams Powershell module. Please use Credential parameter to provide username and password. 
```

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 2]
```diff
@@ -69,6 +64,8 @@ Establishes a Skype for Business Online Remote PowerShell session using multi-fa
 
 ### -UserName
 Specifies the Skype for Business Online administrator account name to be used when prompting for credentials.
+> [!WARNING]
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsOnlineSession.md:67]:
```suggestion
```

**Review by get-itips — CHANGES_REQUESTED** [review]
[!WARNING] or [!NOTE] are not to be used on cmdlet's articles as they break online help. Can you kindly review my suggestion?

File: skype/skype-ps/skype/New-CsOnlineSession.md [hunk 3]
```diff
@@ -179,6 +176,20 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -TeamsEnvironmentName
+Use this setting if your organization is in one of the Teams Government Cloud environments.
+
+Specify "TeamsGCCH" if your organization is in the GCC High Environment. Specify "TeamsDOD" if your organization is in the DoD Environment.
+
+```yaml
+Type: String
```
Comment (get-itips) [inline skype/skype-ps/skype/New-CsOnlineSession.md:185]:
```suggestion
Type: String
Parameter Sets: (All)
```

File: skype/skype-ps/skype/New-CsOnlineSession.md (hunk 1 above)
Comment (dariomws) [inline skype/skype-ps/skype/New-CsOnlineSession.md:68]:
```suggestion
Please Note: UserName has been deprecated and is not available in the Teams Powershell module. Please use the Credential parameter to provide username and password.
```

**Review by dariomws — CHANGES_REQUESTED** [review]
Agree with @get-itips suggested changes.

**Pull request merged**